core/templates/dev/head/base-components/header.template.html

```diff
@@ -10,6 +10,9 @@
 <meta property="og:site_name" content="Oppia">
 <meta property="og:description" content="<%= htmlWebpackPlugin.options.meta.description %>">
 
+<link rel="preload" href="/third_party/generated/webfonts/fa-solid-900.woff2" as="font" type="font/woff2" crossorigin>
+<link rel="preload" href="/third_party/generated/webfonts/fa-brands-400.woff2" as="font" type="font/woff2" crossorigin>
+
```

Comment:
I think these fonts need to be removed from where they were loaded initially.
It should be in a css file.

Reply:
Should be in thirdparty.css

Reply:
I'm not sure what you mean, these are defined just to preload these fonts in order to get them more quicker, because in `third_party.css` we first need to wait for that file to load and then the font is loaded. But I can probably move these preloads to `pages/header_css_libs.html`.

Reply:
Right.
I get it now

Reply:
Moved to `pages/header_css_libs.htm`